modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/mini_cart/EditItemUnitOfMeasure.js

```diff
@@ -429,8 +430,16 @@ const PriceRows = ({price}) => {
 						</span>
 					</PriceRow>
 				</div>
-			) : (
-				<></>
+			)}
+
+			{price && priceOnApplication && (
+				<div className="mini-cart-prices mt-4">
+					<PriceRow>
```

Comment:
Is it correct to don't have the "label" on the left if it's price on application?

Reply:
I thought about this @smottal , but I wasn't sure what the correct "label" would look like. I simply followed our current minicart behavior where the item does not show the price, but rather have Price On Application instead. I created a thread to check with Bernard and David